File: client-java/controller/src/main/java/org/evomaster/client/java/controller/db/DbUnsupportedException.java
package org.evomaster.client.java.controller.db;

public class DbUnsupportedException extends IllegalStateException{

    DbUnsupportedException(SupportedDatabaseType type){
        this(type.toString());
    }

    DbUnsupportedException(String type){
        super("Database type " + type + " is not supported");
    }

}

File: client-dotnet/src/Controller/Controllers/db/DbUnsupportedException.cs

using System;
using Controller.Api;

namespace Controller.Controllers.db
{
    public class DbUnsupportedException : InvalidOperationException
    {

        public DbUnsupportedException(DatabaseType type) : this(type.ToString())
        {
        }

        public DbUnsupportedException(string type) : base("Database type " + type + " is not supported")
        {
        }

    }
}